src/components/WwwFrame/TheHeader.vue

```diff
@@ -53,7 +53,7 @@
 						<router-link
 							v-show="!isVisitor"
 							:id="myKivaMenuId"
-							to="/portfolio"
+							:to="redirectToMyKivaHomepage ? '/mykiva' : '/portfolio'"
```

Comment:
Sorry I didn't call it out specifically, but we want the nav portfolio link to still go to the old portfolio page. This work is to make the homepage MyKiva basically for logged-in users.

Reply:
So with that in mind, we want the header "Kiva" logo to take the user to `/mykiva`, if they are logged in.